uniflow-androidx/src/main/java/io/uniflow/androidx/flow/AndroidDataFlow.kt

```diff
@@ -44,25 +47,23 @@ abstract class AndroidDataFlow(
         defaultState: UIState = UIState.Empty,
         defaultCapacity: Int = Channel.BUFFERED,
         defaultDispatcher: CoroutineDispatcher = UniFlowDispatcher.dispatcher.io()
-) : ViewModel(), DataFlow {
-    private val tag = this.toString()
-    val dataPublisher: LiveDataPublisher = LiveDataPublisher(defaultState)
+) : ViewModel(), DataFlow, DataPublisher {
+    override val tag = this.toString()
+
     private val coroutineScope: CoroutineScope = viewModelScope
-    private val dataStore: UIDataStore = UIDataStore(dataPublisher, defaultState,tag)
-    private val reducer: ActionReducer = ActionReducer(dataStore, coroutineScope, defaultDispatcher, defaultCapacity,tag)
-    private val actionDispatcher: ActionDispatcher
-        get() = ActionDispatcher(viewModelScope, reducer, dataStore, this,tag)
 
-    final override fun getCurrentState() = actionDispatcher.getCurrentState()
-    final override fun <T : UIState> getCurrentStateOrNull(stateClass: KClass<T>): T? = actionDispatcher.getCurrentStateOrNull()
-    final override fun action(onAction: ActionFunction<UIState>): ActionFlow = actionDispatcher.action(onAction)
-    final override fun action(onAction: ActionFunction<UIState>, onError: ActionErrorFunction): ActionFlow = actionDispatcher.action(onAction, onError)
-    final override fun <T : UIState> actionOn(stateClass: KClass<T>, onAction: ActionFunction<T>): ActionFlow = actionDispatcher.actionOn(stateClass, onAction)
-    final override fun <T : UIState> actionOn(stateClass: KClass<T>, onAction: ActionFunction<T>, onError: ActionErrorFunction): ActionFlow = actionDispatcher.actionOn(stateClass, onAction, onError)
+    val defaultDataPublisher = liveDataPublisher(defaultState, "main")
```

Comment:
Can we have a const for `"main"`  also we can remove `default` from  `defaultDataPublisher`, using just `dataPublisher` should be fine :)

Reply:
yes, "main" should be an internal constant. For `defaultDataPublisher` naming, I begin to reply in general comments https://github.com/uniflow-kt/uniflow-kt/pull/56#issuecomment-743046553